### PR TITLE
[Functions] Type and version

### PR DESCRIPTION
**Pull request by justinkaseman**
### Description
Ensures consistent type and version getter for Functions contracts
Cleans up unneeded functions in the Billing Registry to get subscription owner, re-uses getSubscription instead

### CODE REVIEW

**Review by bolekk — APPROVED** [review]
lgtm

File: contracts/src/v0.8/dev/functions/FunctionsBillingRegistry.sol [hunk 1]
```diff
@@ -637,18 +645,6 @@ contract FunctionsBillingRegistry is
     return currentsubscriptionId;
   }
 
-  /**
-   * @notice Gets subscription owner.
-   * @param subscriptionId - ID of the subscription
-   * @return owner - owner of the subscription.
-   */
-  function getSubscriptionOwner(uint64 subscriptionId) external view override returns (address owner) {
```
Comment (KuphJr) [inline contracts/src/v0.8/dev/functions/FunctionsBillingRegistry.sol:645]: Why is this being removed?  Did we confirm we don't rely on this anywhere in the tooling?

**Review by KuphJr — APPROVED** [review]
I just want to confirm we don't rely on getSubscriptionOwner in the tooling.  If so, we just need a tooling PR to update to use getSubscription.

**Review by pinebit — APPROVED** [review]

**Comment by justinkaseman** [comment]
Closing in favor of incoming Functions Router work, which will implement this.

**Pull request closed without merging**